go.mod

```diff
@@ -5,63 +5,25 @@ go 1.16
 require (
```

Comment:
Should we change go version here too?

Reply:
No

Reply:
So we have go 1.16 in devkit and in csi-beremetal go mod, go 1.17 in PR validation and in csi-baremetal-tests go.mod. Why do we need to upgrade go version for tests and why can't update it everywhere?

Reply:
`go help go.mod` 

> The expected language version, set by the go directive, determines
> which language features are available when compiling the module.
> Language features available in that version will be available for use.
> Language features removed in earlier versions, or added in later versions,
> will not be available. Note that the language version does not affect
> build tags, which are determined by the Go release being used.

Reply:
> The basic guideline is fairly simple: a "go" directive 1.N means that the code in that module is permitted to use language features that existed in 1.N even if those features were removed in later releases of the language.

Reply:
Ok, could you please explain why do we need to upgrade go version for csi-baremetal-tests only? Is it works in devkit with go 1.16?

Reply:
There is no "version upgrade", `go 1.N` declare the version as described above, you free to use any version.

Reply:
But we use go compailer 1.16 installed in devkit and declare go 1.17 in go.mod. May we have conflicts?

Reply:
no